.github/workflows/prereq.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: Quick Checker
+
+on:
+  push:
+    branches: [ 'ci-*' ]
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 500
```

Comment:
I still believe that we should pull in the entire repo:
```yaml
fetch-depth: 0
```
It currently takes about 4 minutes, so it's not that bad.